src/sage/plot/complex_plot.pyx

```diff
@@ -561,7 +561,7 @@ def complex_to_cmap_rgb(z_values, cmap='turbo', contoured=False, tiled=False,
     import matplotlib as mpl
 
     if isinstance(cmap, str):
-        cmap = mpl.cm.get_cmap(cmap)
+        cmap = mpl.colormaps.get_cmap(cmap)
```

Comment:
```suggestion
        cmap = mpl.colormaps[cmap]
```
I think this is more idiomatic (and maybe less likely to be deprecated in the future), see:
https://matplotlib.org/stable/api/matplotlib_configuration_api.html#colormaps-and-color-sequences

Reply:
adopted